pkg/virtctl/imageupload/imageupload_test.go

```diff
@@ -249,6 +285,17 @@ var _ = Describe("ImageUpload", func() {
 			Expect(cmd()).NotTo(BeNil())
 		})
 
+		It("uploadProxyURL not configured", func() {
+			testInit(http.StatusOK)
+			cmd := tests.NewRepeatableVirtctlCommand(commandName, "--pvc-name", pvcName, "--pvc-size", pvcSize,
+				"--insecure", "--image-path", imagePath)
+			config, err := cdiClient.CdiV1alpha1().CDIConfigs().Get(configName, metav1.GetOptions{})
+			Expect(err).To(BeNil())
+			config.Status.UploadProxyURL = nil
+			updateCDIConfig(config)
+			Expect(cmd()).NotTo(BeNil())
```

Comment:
Maybe also good to check that the error is the one you expect. Btw. there exists also a nice meaningful `BeNil()` check for errors, namely `NotTo(Succeed())`